packages/yarnpkg-core/sources/formatUtils.ts

```diff
@@ -59,7 +59,7 @@ const chalkOptions = process.env.GITHUB_ACTIONS
     : {level: 0};
 
 export const supportsColor = chalkOptions.level !== 0;
-export const supportsHyperlinks = supportsColor && !process.env.GITHUB_ACTIONS && !isCircleCI;
+export const supportsHyperlinks = supportsColor && !process.env.GITHUB_ACTIONS && !isCircleCI && !process.env.GITLAB_CI;
```

Comment:
Alternatively you can use the `GITLAB` variable from `ci-info`, same as we do for `isCircleCI`

Reply:
Did that, plus updated `ci-info` to do the same with `GITHUB_ACTIONS`

Reply:
Perfect, could do the same here https://github.com/yarnpkg/berry/blob/6c2f8aa589aa6db9e2d18b0f697abec577417e6d/packages/plugin-essentials/sources/commands/install.ts#L97-L98